apps/test/solutions/applab/ec_design.js

```diff
@@ -518,6 +518,10 @@ module.exports = {
         $('#designModeViz .textArea div').first().click();
         assertPropertyRowValue(0, 'id', 'text_area1', assert);
         assert.equal($('#propertyRowContainer textarea').first().val(), 'Text1\n\nText2', 'Text should be written');
+
+        // Reacquire reference to textArea, since it was unmounted when we
+        // selected the screen.
+        textArea = $('#propertyRowContainer textarea').first()[0];
```

Comment:
This actually appears to be an existing "bug" in this test - depending on a particular element persisting across renders in a way that React doesn't guarantee, which broke in the upgrade.  I worry a little that we'll see this sort of problem elsewhere, though in our production React code we rarely depend on this sort of persistence (it's not idiomatic).